net/gce.sh

```diff
@@ -305,13 +298,27 @@ else
   sshPrivateKey="$netConfigDir/id_$prefix"
 fi
 
+defaultZone="$(cloud_DefaultZone)"
+
 case $cloudProvider in
 gce)
+  customMemoryGB="$(cloud_DefaultCustomMemoryGB)"
+  if [[ "$tmpfsAccounts" = "true" ]]; then
+    defaultZone="us-west2-a" # There's some resource limit on us-west1 ATM
+    customMemoryGB=$(( customMemoryGB * 2 ))
```

Comment:
weird to change the memory when specifying this. Can't we already change the memory with `--custom-machine-type`?

Reply:
Kinda.  As it stands, `--custom-machine-type` thing is a bit of a mess when specifying the machine config with individual `--custom-*` flags. Ideally we'd be able to specify all of that stuff discretely instead of ramming it all into one arg's parameter string.  Assuming defaults and doubling that for tmpfs seemed like the quickest path to the feature.  Ideally we'd just sanity check the user-specified machine and all of the tmpfs/swap config would be more granular and we can take it there once there's demand.  For now I just wanted to get something in to ease deployment of the next Kin benchmark cluster

Reply:
Well, the problem with that is that azure/aws don't support adjusting those things individually.

Reply:
Yep! That's why it's gated under GCP for now

Reply:
@sakridge I went ahead and plumbed a `--custom-memory-gb` arg for GCE, then replaced the silent doubling with a warning

https://github.com/solana-labs/solana/pull/14459/commits/3390e5d8fe9c11b22fcaf770fc037adf79e8a2cf